test/DebuggerCommon/step_in_from_interpreted_function_attach.js.dbg.baseline

```diff
@@ -2,19 +2,19 @@
   {
     "callStack": [
       {
-        "line": 27,
+        "line": 29,

```

Comment:
I wish we could arbitrarily ignore certain objects from all our baselines: WScript, Debug, ... (may we attach and examine a special property, e.g. hide object data completely if it has property __hide_baseline__?)